### PR TITLE
Feature user story 198 merge

### PR DESCRIPTION
**Pull request by jrglomar-softvision**

### CODE REVIEW

File: src/main/java/com/csv/communitytrackerjava/service/ProjectServiceImpl.java [hunk 1]
```diff
@@ -77,16 +79,29 @@ public ProjectResponseDTO updateProject(ProjectUpdateDTO projectUpdateDTO, int i
...
     }
-    
-    public ProjectResponseDTO toProjectResponseDTO(String message, ProjectPayloadDTO payloadDTO){
+
+    @Override
+    public Page<ProjectGetPeopleDTO> findPeopleByProjectId(Pageable pageable, Set<Integer> id) throws Exception {
+        List<ProjectGetPeopleDTO> projectList = (
+                projectRepository.findAllByProjectIdIn(id, pageable)
+                        .stream()
+                        .map(projects -> projectMapper.toGetPeopleDTO(projects))
```
Comment (eldrichm-softvision) [inline src/main/java/com/csv/communitytrackerjava/service/ProjectServiceImpl.java]: You can use lambda expression here with declaring projects variables as you're passing the project entity right ahead.

sample. projectMapper::toGetPeopeleDto

File: src/test/java/com/csv/communitytrackerjava/controller/ProjectControllerTestGetPeopleByProjectId.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest
+class ProjectControllerTestGetPeopleByProjectId {
```
Comment (eldrichm-softvision) [inline src/test/java/com/csv/communitytrackerjava/controller/ProjectControllerTestGetPeopleByProjectId.java:33]: Add test method that will test the second page.

File: src/test/java/com/csv/communitytrackerjava/service/ProjectServiceImplTestGetPeopleByProjectId.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class ProjectServiceImplTestGetPeopleByProjectId {
```
Comment (eldrichm-softvision) [inline src/test/java/com/csv/communitytrackerjava/service/ProjectServiceImplTestGetPeopleByProjectId.java:31]: Add test method as well for a multiple page retrieval.

File: src/main/java/com/csv/communitytrackerjava/model/Project.java [hunk 4]
```diff
@@ -29,4 +31,13 @@ public class Project extends BaseAuditClass {
     @Column(name="isactive", nullable = false)
     private Boolean isActive;
 
+    @OneToMany(mappedBy = "projectId", fetch = FetchType.LAZY)
+    private List<People> peoples;
+
+    public Project(Integer projectId, String projectDesc, String projectCode, Boolean isActive) {
```
Comment (eldrichm-softvision) [inline src/main/java/com/csv/communitytrackerjava/model/Project.java:37]: I think it's duplicate on AllArgsConstructor by Lombok?

**Review by eldrichm-softvision — DISMISSED** [review]

File: src/main/java/com/csv/communitytrackerjava/model/Project.java (hunk 4 above)
Comment (jrglomar-softvision) [reply src/main/java/com/csv/communitytrackerjava/model/Project.java:37]: List of people is not included in this constructor

Comment (eldrichm-softvision) [reply src/main/java/com/csv/communitytrackerjava/model/Project.java:37]: Okay, I missed the people what is the class that using the constructor?

**Review by eldrichm-softvision — APPROVED** [review]

**Pull request merged**